HalideGenerator.cmake

```diff
@@ -77,10 +77,10 @@ function(halide_add_generator_dependency)
   # generated library
   if (WIN32)
```

Comment:
It's something that you're about to run lib.exe on, so I think msvc conventions are probably the right thing.